src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1353,6 +1353,7 @@
   "com.minecolonies.coremod.research.limit.onemaxperbranch": "Only one max-level research can be unlocked per branch.",
   "com.minecolonies.coremod.research.limit.immutable": "This research cannot be undone.",
   "com.minecolonies.coremod.research.limit.maxeffect": "MAX",
+  "com.minecolonies.coremod.research.limit.requirement": "Requirement : %s %s",
```

Comment:
Why a space before the colon?

Reply:
Mostly just that it kinda bled together with the tips of the t, but not attached to it in the slightest.

Reply:
maybe a different word than requirement fits better, as we have
Requires: Enchanter level 3 as requirement already. Maybe `Research Cost:` or just `Cost:`